### ISSUE TITLE
Activity session bug: only set completed_at once (do not reset it)

### ISSUE THREAD

**Issue by kriskelly** [empirical-org/Empirical-Core#569]
This code in activity_session.rb sets the completed_at date anytime an activity session gets saved:

```
  def set_completed_at
    return true if state != 'finished'
    self.completed_at = Time.current
  end
```

It should do something like this instead:

```
self.completed_at ||= Time.current
```

**Comment by kriskelly** [comment]
Please include a test case to show that this bug is fixed.

**Comment by marcellosachs** [comment]
@kriskelly fixed with commit 46ca43c61a4c0bea6359b7a38179e4b3a3668017

**Comment by marcellosachs** [comment]
includes spec example in spec/models/activity_session_spec.rb
https://github.com/empirical-org/Empirical-Core/blob/develop/spec/models/activity_session_spec.rb#L183-L191
(not sure why github indenting is weird, its fine if you look at the code in sublime text)
